app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/QuotedMessage.kt

```diff
@@ -31,52 +34,118 @@ import com.wire.android.ui.common.StatusBox
 import com.wire.android.ui.common.colorsScheme
 import com.wire.android.ui.common.dimensions
 import com.wire.android.ui.common.typography
+import com.wire.android.ui.home.conversations.messages.QuotedMessageStyle.COMPLETE
+import com.wire.android.ui.home.conversations.messages.QuotedMessageStyle.PREVIEW
 import com.wire.android.ui.home.conversations.model.QuotedMessageUIData
 import com.wire.android.ui.theme.wireColorScheme
 import com.wire.android.util.ui.UIText
 
 private const val TEXT_QUOTE_MAX_LINES = 7
 
+/**
+ * Indicates whether this QuotedMessage should display all the possible details or not.
+ */
+enum class QuotedMessageStyle {
+    /**
+     * Will attempt to display all possible information in normal size.
+     */
+    COMPLETE,
+
+    /**
+     * Will keep only the most important information, hiding secondary UI elements like:
+     * - Footer with date/time
+     * - Edit time
+     * - Reply Icon
+     *
+     * Images have a reduced size.
+     */
+    PREVIEW
+}
+
 @Composable
 internal fun QuotedMessage(
-    messageData: QuotedMessageUIData
+    messageData: QuotedMessageUIData,
+    style: QuotedMessageStyle = COMPLETE,
+    modifier: Modifier = Modifier,
```

Comment:
Although not being used at the moment, modifier is now being passed all the way down in case it's necessary later.